soc/arm/renesas_smartbond/da1469x/soc.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SOC__H_
+#define _SOC__H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+
+#include <devicetree.h>
+#include <sys/util.h>
+#include "DA1469xAB.h"
+
+#ifdef __cplusplus
+}
+#endif
+
+#include <core_cm33.h> /*!< ARM Cortex-M33 processor and core peripherals */
```

Comment:
this file just seem to need `#include <DA1469xAB.h>`, all others are not required (and we need to fix such indirect dependency at some point, so we can finally remove soc.h on ARM).